src/main.cpp

```diff
@@ -273,13 +273,13 @@ int main ( int argc, char** argv )
                                  argv,
                                  i,
                                  "--centralserver", // no short form
-                                 "--centralserver",
+                                 "--directoryserver",
```

Comment:
I'd add a comment that this is an alternative for compatibility reasons.

Reply:
Done